src/Console/src/Command.php

```diff
@@ -73,27 +73,43 @@ public function setEventDispatcher(EventDispatcherInterface $eventDispatcher): v
     /**
      * Pass execution to "perform" method using container to resolve method dependencies.
      */
-    protected function execute(InputInterface $input, OutputInterface $output): int
+    final protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
I think we should use `@final` annotation instead. Because of BC

Reply:
@roxblnfk Agree, changed to phpdoc